CMakeLists.txt

```diff
@@ -70,6 +70,8 @@ if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/.git)
       string(REGEX REPLACE "(.+)-(.+)-(.+)" "\\2" "COMMITS_SINCE_LAST_RELEASE" "${GIT_DESCRIBE}")
       if(NOT "${COMMITS_SINCE_LAST_RELEASE}" EQUAL "0")
         string(REGEX REPLACE "(.+)-(.+)-(.+)" "\\1.r\\2.\\3" "DISPLAY_VERSION" "${GIT_DESCRIBE}")
+      else()
+        string(REGEX REPLACE "([0-9]+\\.[0-9]+)\\..+" "v\\1" "MINOR_VERSION" "${GIT_DESCRIBE}")
```

Comment:
MINOR VERSION seems a bad name choice. It is confusing. Suppose a version is 6.7.2 the term MINOR VERSION of above version is just 7. While a correct name could be MAJOR_MINOR_VERSION which could be interpreted as v6.7 (includes both major and minor number).

Reply:
I think "6" is a "major version number", "v6" is a "major version", "7" is a "minor version number", and "v6.7" is a "minor version".

Reply:
You can't say "7" in "6.7.2" is a "version".

Reply:
Fine then!